themes/yt/template.html

```diff
@@ -58,7 +58,18 @@
   }
 </style>
 
-<media-controller breakpoints="md:480">
+<media-controller
+  breakpoints="md:480"
+  nodefaultstore="{{nodefaultstore}}"
+  defaultsubtitles="{{defaultsubtitles}}"
+  defaultduration="{{defaultduration}}"
+  gesturesdisabled="{{disabled}}"
+  hotkeys="{{hotkeys}}"
+  nohotkeys="{{nohotkeys}}"
+  audio="{{audio}}"
```

Comment:
I don't know about this one. It's not really meant for audio and I haven't tested it for that use case, so I think I'd rather exclude it.